tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/frontendutils/JsonReader.java

```diff
@@ -0,0 +1,38 @@
+package ca.bc.gov.open.jagefilingapi.qa.frontendutils;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Map;
+
+public class JsonReader {
+
+    public static Object[][] searchJsonElement(JsonArray jsonArray, int totalDataRow, int totalColumnEntry) {
+        Object[][] matrix = new Object[totalDataRow][totalColumnEntry];
+        int i = 0;
```

Comment:
I think we can define these on one line? also I'm a sucker for descriptive counter names =P

Reply:
Refactored. Thanks.